src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -1184,10 +1184,8 @@ bool AudioUnitEffect::RealtimeAddProcessor(
    slave->SetBlockSize(mBlockSize);
    slave->SetSampleRate(sampleRate);
 
-   if (!CopyParameters(mUnit.get(), slave->mUnit.get()))
-   {
+   if (!slave->StoreSettings(GetSettings(settings)))
```

Comment:
I may be missing something here but: is this really equivalent to what happened before? this sets the slave with input parameter `settings` - these in principle could be different from the settings currently in mUnit